test/functional/eval/fnamemodify_spec.lua

```diff
@@ -0,0 +1,45 @@
+local helpers = require('test.functional.helpers')(after_each)
+local clear = helpers.clear
+local eq = helpers.eq
+local iswin = helpers.iswin
+local fnamemodify = helpers.funcs.fnamemodify
+local command = helpers.command
+local write_file = helpers.write_file
+
+describe('fnamemodify()', function()
+  setup(function()
+    write_file('Xtest-fnamemodify.txt', [[foobar]])
+  end)
+
+  before_each(clear)
+
+  teardown(function()
+    os.remove('Xtest-fnamemodify.txt')
+  end)
+
+  it('works', function()
+    if iswin() then
+      local drive_f = io.popen('for %P in (%CD%) do @echo %~dP', 'r')
+      local drive = string.gsub(drive_f:read('*a'), '[\n\r]', '')
```

Comment:
on master ( https://github.com/neovim/neovim/commit/499c9a15531b7a0e9736a395e8d401ceab3d24d2 ) there is now `helpers.pathroot()`, can that be used instead?

Reply:
Yes. I can strip out the pathsep and append a forward slash for `shellslash`.